samples/security/spm/spm_nspe/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _spm_psoc6:
+
+SPM base NS environment example for PSoC6  CM4
+###########
+
+Overview
+********
+A simple SPM base NS environment example for PSoC6  CM4.
+Needs Secure part on CM0+
+
+CM4(spm_nspe) sends debug log to Uart Port5 which is available through USB connection with the PSoC6 kit.
+CM0+(spm_spe) send debug messages to UART port 12(pin0(RX)/pin1(TX)) � connect external UART to UASB convertor to these pins on kit 
```

Comment:
This file contains non-ACSII/non-UTF-8 characters (that `–`).   Please use an editor to replace  window-1252 characters with their ASCII equivalents.  (Windows editors are notorious for changing things line single ``'`` and double quotes ``"`` into "smart" quotes.)  You can use the linux comand ``file --mime-encoding | grep "unknown"`` to find these files.

Reply:
@d3zd3z Here's the non-UTF8 encoding problem (an en-dash)

Reply:
Already fixed. It seems you have an older version of the tree